ci-operator/config/redhat-openshift-ecosystem/msp/redhat-openshift-ecosystem-cvp-ocp-4.8.yaml

```diff
@@ -0,0 +1,37 @@
+releases:
+  latest:
+    release:
+      channel: stable
+      version: "4.8"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: msp-common-claim
```

Comment:
There would be 4 configurations for msp cluster pools 2 in each file . 
Make sure the labels match with msp clusterpools.